apps/web/public/static/locales/en/common.json

```diff
@@ -585,101 +385,51 @@
   "url": "URL",
   "hidden": "Hidden",
   "readonly": "Readonly",
-  "one_time_link": "One-time link",
-  "plan_description": "You're currently on the {{plan}} plan.",
-  "plan_upgrade_invitation": "Upgrade your account to the PRO plan to unlock all of the features we have to offer.",
-  "plan_upgrade": "You need to upgrade your plan to have more than one active event type.",
-  "plan_upgrade_teams": "You need to upgrade your plan to create a team.",
-  "plan_upgrade_instructions": "You can <1>upgrade here</1>.",
   "event_types_page_title": "Event Types",
   "event_types_page_subtitle": "Create events to share for people to book on your calendar.",
   "new": "New",
-  "new_event_type_btn": "New event type",
   "new_event_type_heading": "Create your first event type",
   "new_event_type_description": "Event types enable you to share links that show available times on your calendar and allow people to make bookings with you.",
-  "new_event_title": "Add a new event type",
-  "new_team_event": "Add a new team event type",
```

Comment:
This is removed from english translation file but not from other languages files